network-api/networkapi/campaign/views.py

```diff
@@ -125,6 +125,7 @@ def petition_submission(request, petition):
         "email": request.data['email'],
         "email_subscription": request.data['newsletterSignup'],
         "source_url": request.data['source'],
+        "page_locale": request.data['locale'],
     }
 
     if petition.requires_country_code:
```

Comment:
@Pomax, does this send the selected country to the backend as long as the petition creator marks `requires_country_code` in `Petition` class `true`?

Reply:
Yes, if the petition is set to require a country code, then this block of code makes sure to either get the value from the form passed on, or generate an error about the face that this required field is missing.